store/src/java/com/zimbra/qa/unittest/TestUserServlet.java

```diff
@@ -74,29 +74,39 @@
 import com.zimbra.cs.mailbox.Mailbox;
 import com.zimbra.cs.util.JMSession;
 
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
 
-public class TestUserServlet
-extends TestCase {
 
+public class TestUserServlet{
+	
+    @Rule
+    public TestName testInfo = new TestName();
+    private static String USER_NAME = null;
     private static final String NAME_PREFIX = TestUserServlet.class.getSimpleName();
-    private static final String USER_NAME = "user1";
     private static String id1;
     private static String id2;
     private String originalSanitizeHtml;
 
-    @Override
+    @Before
     public void setUp()
     throws Exception {
+        String prefix = NAME_PREFIX + "-" + testInfo.getMethodName() + "-";
+        USER_NAME = prefix + "user";		
```

Comment:
Please remove tabs at the end of this line.  I use eclipse which has the capability of showing white space characters.  If your editor allows that, I recommend that you switch it on.